### ISSUE TITLE
Search bar

### ISSUE THREAD

**Issue by snug** [light-launcher/Light-Android-Launcher#25]
In the all apps screen, it would be nice to have a discreet search bar so that people can search through the apps list instead of manually scrolling.

**Comment by jancborchardt** [comment]
To make it discreet, it would be best if it would be above the view and only visible if you scroll up. Could even automatically focus and open the keyboard if you scroll up. Then you don’t need to move your finger so far up.

(If we do that, we need to make sure that when you scroll down and then scroll up, it doesn’t trigger search directly, but only after scrolling up again. If possible.)

**Comment by postapczuk** [comment]
Wouldn't be simpler to create scroll with alphabet and scroll to chosen letter? I'm really not a fan of filters of the app list, because all I was doing on such filters was writing the first letter - as the app was available on the view after that operation. I hated that because phone needed to launch keyboard and it's animation then to write letter...

**Comment by jancborchardt** [comment]
Scroll with alphabet seems good too as it does not require typing! That will not work then for any other sorting than alphabetic though (ref #15 sort by frequency).

**Comment by taylor-josh** [comment]
this appears to have been handled in the most recent release - 1.8.0
